### PR TITLE
Release/v0.10.0

### PR DESCRIPTION
**Pull request by bblaszkow06**

### CODE REVIEW

File: lib/membrane_mp3_mad_plugin/decoder.ex [hunk 1]
```diff
@@ -74,21 +66,21 @@ defmodule Membrane.MP3.MAD.Decoder do
         {:ok, {buffer, Enum.reverse(acc)}}
 
       {:error, {:recoverable, bytes_to_skip}} ->
-        warn("Skipping malformed frame (#{bytes_to_skip} bytes)")
+        Logger.warn("Skipping malformed frame (#{bytes_to_skip} bytes)")
         <<_used::binary-size(bytes_to_skip), new_buffer::binary>> = buffer
 
         case acc do
           [{:event, _} | _] ->
```
Comment (mat-hek) [inline lib/membrane_mp3_mad_plugin/decoder.ex:73]: seems like an unnamed underscore, shouldn't credo complain?

File: lib/membrane_mp3_mad_plugin/decoder.ex [hunk 2]
```diff
@@ -3,17 +3,18 @@ defmodule Membrane.MP3.MAD.Decoder do
...
+  require Membrane.Logger
+
   alias Membrane.Caps.Audio.{Raw, MPEG}
   alias __MODULE__.Native
-  alias Membrane.Buffer
-  use Membrane.Log
+  alias Membrane.{Buffer, Logger}
 
-  def_input_pad :input, demand_unit: :buffers, caps: [:any, MPEG]
+  def_input_pad :input, demand_mode: :auto, caps: [:any, MPEG]
```
Comment (mat-hek) [inline lib/membrane_mp3_mad_plugin/decoder.ex:12]: I think it should accept RemoteStream instead of any

**Review by mat-hek — CHANGES_REQUESTED** [review]

File: lib/membrane_mp3_mad_plugin/decoder.ex (hunk 1 above)
Comment (bblaszkow06) [reply lib/membrane_mp3_mad_plugin/decoder.ex:73]: Well, it complained about line 86, I'm not sure why this is ignored, but it is :shrug:

File: lib/membrane_mp3_mad_plugin/decoder.ex (hunk 2 above)
Comment (bblaszkow06) [reply lib/membrane_mp3_mad_plugin/decoder.ex:12]: Well, this means the file element should send it - right now it doesn't and defines output caps as `:any`. I'll create an issue for that

**Pull request merged**